openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -144,19 +149,23 @@ export const CustomTooltip = (props: DataInsightChartTooltipProps) => {
     active,
     payload = [],
     valueFormatter,
+    showFullTimestamp = false,
```

Comment:
Let's have a axisFormatter to allow any time rendering instead restricting it to certain fixed typed